server/sendEthernet.go

```diff
@@ -0,0 +1,93 @@
+package server
+
+//function from https://gist.github.com/corny/5e4e3f8e6f2395726e46c3db9db17f12#file-dhcp_discover-go
+import (
+	"encoding/binary"
+	"net"
+	"time"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcap"
```

Comment:
For reference, how much of these are unix (or linux) only ? I believe we already have a couple parts where the code is linux-only but it'd be nice to document it as much as possible for if/when we try to broaden supported platforms

Reply:
According https://pkg.go.dev/github.com/google/gopacket@v1.1.19/pcap, you need to install winpcap or npcap to use it on Windows, but I didn't try.